File: app/src/main/java/com/example/achalker/starbuzz/StoreCategoryActivity.java
package com.example.achalker.starbuzz;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

public class StoreCategoryActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_store_category);

        ArrayAdapter<Store> listAdapter = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, Store.stores);
        ListView listStores = (ListView) findViewById(R.id.list_stores);
        listStores.setAdapter(listAdapter);
    }
}

File: app/src/main/java/com/example/achalker/starbuzz/StoreCategoryActivity.kt
package com.example.achalker.starbuzz

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import android.view.View
import android.widget.ArrayAdapter
import android.widget.ListView
import com.example.achalker.starbuzz.model.Store

class StoreCategoryActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_store_category)
        val listAdapter = ArrayAdapter(this, R.layout.list_item, Store.stores.map { it.name })
        val listStores = findViewById<View>(R.id.list_stores) as ListView
        listStores.adapter = listAdapter
    }
}